spinnaker/spinnaker_testing/spinnaker_test_scenario.py

```diff
@@ -378,6 +378,8 @@ def __update_bindings_with_subsystem_configuration(self, agent):
     it is testing.
     """
     # pylint: disable=bad-indentation
+    if agent is None:
```

Comment:
This case occurs if we don't have/want an agent that is making calls to Spinnaker (as is the case with the quota test).